### ISSUE TITLE
Bug : Error handling an untranslated URL

### ISSUE THREAD

**Issue by Gmulti** [weglot/weglot-php#34]
**Is this a BUG REPORT or FEATURE REQUEST?**: BUG REPORT

**What happened**:
When you want to use the URL object with an untranslated URL and a basePrefix, this does not happen as expected.

**How to reproduce it (some code is appreciated)**:
```php
$url = new Url(
	'http://weglotmultiv2.local/othersite/',
	'en',
	['fr'],
	'/othersite'
);

var_dump($url->currentRequestAllUrls());
/*
 array (size=2)
  'en' => string 'http://weglotmultiv2.local/othersite/othersite/' (length=47)
  'fr' => string 'http://weglotmultiv2.local/othersite/fr/othersite/' (length=50)
*/
```

**Anything else we need to know?**:
The result of this function should be:
```php
/*
 array (size=2)
  'en' => string 'http://weglotmultiv2.local/othersite/'
  'fr' => string 'http://weglotmultiv2.local/othersite/fr/'
*/
```

**Comment by Korbeil** [comment]
The trailing `/` is causing problem to detect base URL.

**Comment by Korbeil** [comment]
Fixed in `0.4.2`
